### PR TITLE
ENH Add MFA docs

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-mfa/issues/531

### CODE REVIEW

File: sources-user.js
Comment (GuySartorelli) [inline sources-user.js:29]:
```suggestion
```
MFA is already in this file, so this is a duplicate entry.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: sources-user.js
Comment (emteknetnz) [reply sources-user.js]: Removed

**Review by GuySartorelli — APPROVED** [review]
FYI this, https://github.com/silverstripe/doc.silverstripe.org/pull/294, and https://github.com/silverstripe/doc.silverstripe.org/pull/295 will conflict with each other - you might want to combine them into a single PR since I won't merge any of them until all the docs PRs are merged anyway.

But looks good to me.

**Comment by GuySartorelli** [comment]
Conflict to resolve. I strongly recommend combining these three PRs into one.

**Comment by emteknetnz** [comment]
Combined

**Comment by GuySartorelli** [comment]
Netlify is complaining:

> Error processing Markdown file /opt/build/repo/.cache/gatsby-source-git/docs--5--optional_features/mfa/docs/en/02_creating-new-mfa-methods/01_creating-mfa-method-frontend.md:
5:02:27 PM:       incomplete explicit mapping pair; a key node is missed; or followed by a non-tabulated empty line at line 2, column 33:
5:02:27 PM:`     title: Creating a new MFA method: front-end`
5:02:27 PM:`                                     ^`

I think it doesn't like having a colon in a metadata title.

**Comment by emteknetnz** [comment]
Have created PR to change colon to dash https://github.com/silverstripe/silverstripe-mfa/pull/534

**Comment by GuySartorelli** [comment]
Netlify is appeased.
[deploy summary](https://app.netlify.com/sites/ss-docs/deploys/6615af2f39e74e3de1bc01c8)
[new preview](https://deploy-preview-292--ss-docs.netlify.app/en/5/optional_features/)

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**
